src/containers/Tenant/utils/schema.ts

```diff
@@ -41,6 +43,49 @@ export const mapPathTypeToNavigationTreeType = (
 
 // ====================
 
+const pathSubTypeToEntityName: Record<EPathSubType, string | undefined> = {
+    [EPathSubType.EPathSubTypeSyncIndexImplTable]: 'Secondary Index Table',
+    [EPathSubType.EPathSubTypeAsyncIndexImplTable]: 'Secondary Index Table',
+
+    [EPathSubType.EPathSubTypeStreamImpl]: undefined,
+    [EPathSubType.EPathSubTypeEmpty]: undefined,
+};
+
+const pathTypeToEntityName: Record<EPathType, string | undefined> = {
+    [EPathType.EPathTypeInvalid]: undefined,
+
+    [EPathType.EPathTypeSubDomain]: 'Database',
+    [EPathType.EPathTypeExtSubDomain]: 'Database',
+
+    [EPathType.EPathTypeDir]: 'Directory',
+    [EPathType.EPathTypeTable]: 'Table',
+    [EPathType.EPathTypeTableIndex]: 'Secondary Index',
+    [EPathType.EPathTypeColumnStore]: 'Tablestore',
+    [EPathType.EPathTypeColumnTable]: 'Columntable',
+    [EPathType.EPathTypeCdcStream]: 'Changefeed',
+    [EPathType.EPathTypePersQueueGroup]: 'Topic',
+};
+
+export const mapPathTypeToEntityName = (
+    type?: EPathType,
+    subType?: EPathSubType,
+): string | undefined =>
+    (subType && pathSubTypeToEntityName[subType]) || (type && pathTypeToEntityName[type]);
+
+// ====================
+
+const databaseTypeToDBName: Record<ETenantType, string | undefined> = {
+    [ETenantType.UnknownTenantType]: 'Database',
+    [ETenantType.Domain]: 'Cluster Root',
+    [ETenantType.Dedicated]: 'Dedicated Database',
+    [ETenantType.Shared]: 'Shared Database',
+    [ETenantType.Serverless]: 'Serverless Database',
+};
+
+export const mapDatabaseTypeToDBName = (type?: ETenantType) => type && databaseTypeToDBName[type];
```

Comment:
what about i18n?

Reply:
I don't really like the idea of translating special terms. Even if we decide to translate them, we should discuss correct naming together

Reply:
If they have translation in the docs, I believe they should be translated in UI. If not, then discussable. As for coming up with the naming — yes, of course we should decide with the team.